### PR TITLE
[STOS-87] STOS API mock

### PR DESCRIPTION
**Pull request by lukrzak**

### CODE REVIEW

File: stos_mock/stos.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from random import randint
+from flask import Flask, jsonify, send_file
+
+app = Flask(__name__)
+
+
+@app.route("/tasks", methods=['GET'])
+def tasks():
+    randomized_task: dict = {
+        "id_studenta": randint(1, 100),
```
Comment (tpalinski) [inline stos_mock/stos.py]: Possibly translate to english

File: stos_mock/stos.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "id_zadania": randint(1, 100),
+        "id_plikow": [randint(1, 20) for _ in range(randint(1, 5))]
+    }
+
+    return jsonify(randomized_task), 200
+
+
+@app.route("/files/<int:file_id>", methods=['GET'])
```
Comment (tpalinski) [inline stos_mock/stos.py]: IMO string is better

File: stos_mock/stos.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    return send_file("data.txt", as_attachment=True, mimetype="text/plain"), 200
+
+
+@app.route("/tasks/<int:task_id>", methods=['POST'])
+def task(task_id: int):
+    return jsonify({"result": "uploaded " + str(task_id)}), 200
+
+
+app.run(host='0.0.0.0', port=5000)
```
Comment (tpalinski) [inline stos_mock/stos.py]: Probably good to move port to .env, more convenient

**Review by tpalinski — COMMENTED** [review]
Nice, small changes and will be miodzio 👌🏻

**Comment by lukrzak** [comment]
STOS API mock according to the [JIRA](https://tpalinski.atlassian.net/browse/STOS-87) task

**Review by tpalinski — APPROVED** [review]
LGTM

**Review by AdamNiesiobedzki — COMMENTED** [review]
dobre api

**Pull request merged**